<div class="form-actions pull-right">
  <%= link_to t('.back', :default => t("helpers.links.back")),
              rails_exception_handler_admin.error_messages_path, :class => 'btn'  %>
  <%= link_to t('rails_exception_handler.resolve'),
              rails_exception_handler_admin.error_message_path(@error_message),
              :method => 'delete',
              :data => { :confirm => t('.confirm', :default => t("helpers.links.confirm", :default => 'Are you sure?')) },
              :class => 'btn btn-danger' %>
</div>
<div class="page-header">
  <h1 class="exception-title"><%= @error_message.class_name %> - <%= @error_message.message %></h1>
</div>

<dl class="dl-horizontal">
  <dt><strong><%= t('rails_exception_handler.app_name') %>:</strong></dt>
  <dd><%= @error_message.app_name || '&nbsp;'.html_safe %></dd>
  <dt><strong><%= t('rails_exception_handler.doc_root') %>:</strong></dt>
  <dd><%= @error_message.doc_root || '&nbsp;'.html_safe %></dd>
  <dt><strong><%= t('rails_exception_handler.user_info') %>:</strong></dt>
  <dd><%= @error_message.user_info || '&nbsp;'.html_safe %></dd>
  <dt><strong><%= t('rails_exception_handler.target_url') %>:</strong></dt>
  <dd><%= @error_message.target_url || '&nbsp;'.html_safe %></dd>
  <dt><strong><%= t('rails_exception_handler.referer_url') %>:</strong></dt>
  <dd><%= @error_message.referer_url || '&nbsp;'.html_safe %></dd>
  <dt><strong><%= t('rails_exception_handler.params') %>:</strong></dt>
  <dd><%= @error_message.params %></dd>
  <dt><strong><%= t('rails_exception_handler.user_agent') %>:</strong></dt>
  <dd><%= @error_message.user_agent || '&nbsp;'.html_safe %></dd>
  <dt><strong><%= t('rails_exception_handler.trace') %>:</strong></dt>
  <dd><%= raw @error_message.trace.try(:split, "\n").try(:join, "<br/>") || '&nbsp;'.html_safe %></dd>
</dl>
